src/client/sandbox/node/index.js

```diff
@@ -37,6 +39,19 @@ export default class NodeSandbox extends SandboxBase {
         }
     }
 
+    _parseDocumentCharset () {
+        var metaCharset = NativeMethods.querySelector.call(document, '.' + CONST.SHADOW_UI_CHARSET_CLASSNAME);
+
+        if (metaCharset) {
+            metaCharset.parentNode.removeChild(metaCharset);
```

Comment:
Anyway, I beliveve it's a bad idea to remove it. We can do it for any our resource, but we don't, because it's useful for the testing purposes.